tests/test_pointset.py

```diff
@@ -106,3 +106,13 @@ def test_filters_return_pointset(sphere):
     pointset = sphere.cast_to_pointset()
     clipped = pointset.clip()
     assert isinstance(clipped, pyvista.PointSet)
+
+
+@pytest.mark.parametrize(
+    "force_float,expected_data_type", [(False, vtk.VTK_LONG_LONG), (True, vtk.VTK_FLOAT)]
+)
+def test_pointset_force_float(force_float, expected_data_type):
+    np_points = np.array([[1, 2, 3]])
+    pset = pyvista.PointSet(np_points, force_float=force_float)
+
+    assert pset.points.GetDataType() == expected_data_type
```

Comment:
Could we do these tests in numpy land, checking `pset.points.dtype`? The issue is probably that "int" numpy arrays are `int32` on (even 64-bit) windows, and `int64` on any sane 64-bit system. Using VTK types adds an additional layer of ambiguity. I'd recommend creating `np_points` with an explicit `dtype=np.int64)` and then checking for the same dtype in the `assert`. This should fix platform-dependence.

Reply:
Good point, please see #3396 which is what this merge request should have been originally be about.